sys-fs/ncdu/ncdu-1.14.1.recipe

```diff
@@ -15,8 +15,8 @@ SECONDARY_ARCHITECTURES="x86"
 PROVIDES="
 	ncdu$secondaryArchSuffix = $portVersion
 	cmd:ncdu$secondaryArchSuffix = $portVersion
- 	commandBinDir = $binDir
 	"
+
```

Comment:
There shouldn't be a space between PROVIDES and REQUIRES. Please remove the extra space here on line 19.

Reply:
@Ra-V2 It seems you have applied this change before I reviewed your previous changes. You can ignore this suggestion as it is outdated.